Rakefile

```diff
@@ -10,15 +10,25 @@ include FileUtils
 
 KARMA_COMMAND = 'node_modules/.bin/karma'
 
-task :karma do
-  puts
-  puts 'Running JavaScript Karma specs'
-  puts
+def described_task(name, description:)
+  task name do
+    puts
+    puts "Running #{description}"
+    puts
+    yield
+    puts
+  end
+end
 
+described_task :karma, description: 'Javascript Karma specs' do
   fail 'ERROR: karma is not installed' unless File.exist? KARMA_COMMAND
   sh "#{KARMA_COMMAND} start spec/javascripts/karma.conf.js --single-run=true"
+end
 
-  puts
+described_task :jshint, description: 'JSHint' do
+  sh 'node_modules/.bin/jshint app/assets/javascripts spec/javascripts' do
```

Comment:
not sure about factoring it out — but you're right it should definitely be there